### PR TITLE
Running Eslint against test code

### PR DESCRIPTION
**Pull request by glauco**
Up to now, we were not running lint on any test file.
This commit applies lint rules to test files and fixes the problems that
arose from this.

### CODE REVIEW

File: test/fetcher/canVariablesFetcher_spec.js [hunk 1]
```diff
@@ -57,24 +60,24 @@ describe('CanVariablesFetcher', () => {
       },
       timeout: config.TIMEOUT
     };
-    respondWithSuccess(httpClient(canVariableRequest), telemetryResponse);
+    helpers.respondWithSuccess(httpClient(canVariableRequest), telemetryResponse);
 
     canVariablesFetcher.fetchByEquipmentId(
-        ['equipment-id-1', 'equipment-id-2'],
-        mockedAuthorizationBearer
-        ).then((response) => {
```
Comment (bltavares) [inline test/fetcher/canVariablesFetcher_spec.js]: 👍

**Comment by bltavares** [comment]
I like most of the changes.

I don't like that we are testing the same controller from two different files. Is this a smell to split the controller itself?

One other options is to instead of increasing the global file line limit, we could set it to be suppressed on that test with a comment.

**Comment by waldemarnt** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

**Comment by bltavares** [comment]
LGTM

**Pull request merged**